WatchFaceAlphaKotlin/app/src/main/java/com/example/android/wearable/alpha/editor/WatchFaceConfigActivity.kt

```diff
@@ -105,13 +107,6 @@ class WatchFaceConfigActivity : ComponentActivity() {
         binding.minuteHandLengthSlider.isEnabled = true
     }
 
-    override fun onDestroy() {
-        stateHolder.onCleared()
```

Comment:
Ah, we still need to call `onCleared` here, right? Otherwise nothing will call that for the `StateHolder`.
There might be a more composable version here where we give the state holder a `Lifecycle` reference to clear itself up.

Reply:
oh geez, sorry, thanks, good catch, I need that to close the editor session.